diff --git a/node_modules/@unimodules/core/android/build.gradle b/node_modules/@unimodules/core/android/build.gradle
index e12c9bb..d8366ea 100644
--- a/node_modules/@unimodules/core/android/build.gradle
+++ b/node_modules/@unimodules/core/android/build.gradle
@@ -1,6 +1,6 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 
 group = 'org.unimodules'
 version = '7.1.2'
